pkgs/tools/misc/t/Gemfile

```diff
@@ -1,3 +1,4 @@
 source "https://rubygems.org"
 
 gem 't'
+gem 'twitter', '~> 6.1.0'
```

Comment:
Can we add a reference to https://github.com/sferik/t/issues/402 so we know when we can remove this in future?